olive/passes/onnx/perf_tuning.py

```diff
@@ -171,6 +180,18 @@ def tune_onnx_model(perf_tuning_pass_ep, model, data_root, config):
     return optimized_model
 
 
+def rocm_tuning_enable(provider, provider_options):
+    assert provider == "ROCMExecutionProvider", "provider should be ROCMExecutionProvider"
+    # Please refer to the following links for the config or ROCMExecutionProvider
+    # https://github.com/microsoft/onnxruntime/blob/71657d1eb8b0a24a4b6584d9e904506a0b4e1521/onnxruntime/core/providers/rocm/rocm_execution_provider_info.cc#L24C1-L25
+    provider_options["tunable_op_enable"] = True
```

Comment:
Just double check, following the link you attached, the accept value type is `char`. Does the ort python API make the transformation happen?
```c++
constexpr const char* kTunableOpEnable = "tunable_op_enable";
constexpr const char* kTunableOpTuningEnable = "tunable_op_tuning_enable";
```

Reply:
![Uploading image.png…]()

Reply:
the bool will be converted to 1/0 in c++ level.